python/sdk/prelude_sdk/controllers/build_controller.py

```diff
@@ -102,6 +105,24 @@ def create_threat(self, name, published, threat_id=None, source_id=None, source=
             return res.json()
         raise Exception(res.text)
 
+    @verify_credentials
+    def create_threat_from_directory(self, directory: str, **kwargs):
```

Comment:
light/light-medium opinion: this belongs in the cli (or a cli utils file) more than the sdk since it relies on a particular directory structure that our threat_intel generates